src/plugins/vis_builder/public/application/components/data_tab/index.tsx

```diff
@@ -77,6 +79,17 @@ export const DataTab = () => {
       }
 
       const panelGroups = Array.from(schemas.all.map((schema) => schema.name));
+      // Check schema order
+      if (destinationSchemaName === 'split') {
+        const validationResult = validateAggregations(aggProps.aggs, schemas.all);
```

Comment:
nit: We are only using split schema during validation we can only pass that schema instead of passing in all the schemas

Reply:
Yeah currently this is an special schema check for pie. It is not appliable for other schemas.